### PR TITLE
omstdout: rewrite documentation

### PR DESCRIPTION
**Pull request by PascalWithopf**
Module documentation was rewritten in the style.
Additionally parameter with the new config style were added.

closes https://github.com/rsyslog/rsyslog-doc/issues/547

### CODE REVIEW

**Comment by rgerhards** [comment]
@deoren The question @PascalWithopf raises in regard to ```template``` is a very good one. It's actually a problem with almost all of the actions. In obsolete legacy format, the template is (usually ;-)) given by appending ```;TemplateName``` to the action parameters - so, yes, you can set it, but, no, it's not a real parameter.

I admit I do not know yet how to handle this smartly inside the table...

**Comment by deoren** [comment]
It's been a hectic day, but I will try to look at this later today when I have some time.

**Comment by deoren** [comment]
> @rgerhards: @deoren The question @PascalWithopf raises in regard to template is a very good one.

I agree, that is a good question and not necessarily one that has an easy answer.

> @rgerhards: It's actually a problem with almost all of the actions. In obsolete legacy format, the template is (usually ;-)) given by appending ;TemplateName to the action parameters - so, yes, you can set it, but, no, it's not a real parameter.
> 
> I admit I do not know yet how to handle this smartly inside the table...

Maybe it's worth spinning off a separate issue just for this. It's something that many (all?) of the other output module docs will need to cover eventually, so maybe it's good to handle that decision in one place.

File: source/configuration/modules/omstdout.rst [hunk 1]
```diff
@@ -16,21 +19,96 @@ the quality of this module as we do not expect it to be used in real
...
+   module(load="omstdout")
+   action(type="omstdout")
+
+
+Example 2
+---------
+
+The following sample will write syslog messages to stdout, using a template
+and input from imtcp.
+
```
Comment (deoren) [inline source/configuration/modules/omstdout.rst]: I'm probably overlooking it, but how does imtcp factor into the example?

Comment (PascalWithopf) [reply source/configuration/modules/omstdout.rst]: Sry, I took imtcp out of the example to keep it as short as possible but forgot to change the description

**Comment by deoren** [comment]
refs: #456, #451

File: source/configuration/modules/omstdout.rst [hunk 2]
```diff
@@ -16,21 +19,95 @@ the quality of this module as we do not expect it to be used in real
...
-Note: configuration directive names are case-insensitive.
+Configuration
+=============
+
+.. note::
+
+   Parameter names are case-insensitive.
+
+
+Module Parameter
```
Comment (deoren) [inline source/configuration/modules/omstdout.rst]:
```diff
-Module Parameter
+Module Parameters
```

Should this be plural?

File: source/configuration/modules/omstdout.rst [hunk 3]
```diff
@@ -16,21 +19,95 @@ the quality of this module as we do not expect it to be used in real
...
+----------------
+
+none
+
+
+Action Parameter
```
Comment (deoren) [inline source/configuration/modules/omstdout.rst]: Same here.

**Pull request merged**